### PR TITLE
*: populate and deploy etcd.conf

### PR DESCRIPTION
**Pull request by hexfusion**
This rounds the corners on static assets for etcd.

* remove etcd-config-file flag

### CODE REVIEW

File: pkg/cmd/render/render.go [hunk 1]
```diff
@@ -9,17 +9,18 @@ import (
...
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	yaml "gopkg.in/yaml.v2"
+	"k8s.io/klog"
 )
 
 const (
 	bootstrapVersion = "v4.2.0"
+	etcdConfigDir    = "/etc/etcd"
 )
```
Comment (hexfusion) [inline pkg/cmd/render/render.go]: this needs to be configurable.

File: pkg/cmd/render/render.go [hunk 2]
```diff
@@ -215,8 +222,25 @@ func (r *renderOpts) Run() error {
 	return WriteFiles(&r.generic, &renderConfig.FileConfig, renderConfig)
 }
 
+// parseStaticTemplateFile takes a path to a yaml template file returns a populates File struct.
+func parseStaticTemplateFile(path string) (*File, error) {
```
Comment (hexfusion) [inline pkg/cmd/render/render.go:220]: s/populates/populated

File: pkg/cmd/render/render.go [hunk 3]
```diff
@@ -215,8 +222,25 @@ func (r *renderOpts) Run() error {
...
+	if err := yaml.Unmarshal(data, file); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal file into struct: %v", err)
+	}
+	return file, nil
+}
+
 func populateFileData(files []StaticFile) ([]StaticFile, error) {
 	for i, file := range files {
+		if file.source == "" {
+			continue
```
Comment (hexfusion) [inline pkg/cmd/render/render.go:238]: add godoc around this.

**Review by beekhof — APPROVED** [review]
/lgtm

**Pull request merged**